lib/jsdp-value-converter.js

```diff
@@ -0,0 +1,94 @@
+var JuttleMoment = require("juttle/lib/moment").JuttleMoment;
+var moment = require("moment");
+var _ = require("underscore");
+
+var dateConverter = {
+    isConvertibleToJSDP: function(value) {
+        return value instanceof JuttleMoment && value.moment;
+    },
+    isConvertibleToJuttle: function(value) {
```

Comment:
The term ToJuttle is a bit confusing to me. Isn't this really converting to juttle-compatible javascript types? Maybe we could use juttle compatible or something instead.